### ISSUE TITLE
Failed to run "train.py" due to missing "model_params" folder

### ISSUE THREAD

**Issue by pennymax** [thomasmesnard/DeepMind-Teaching-Machines-to-Read-and-Comprehend#3]
It failed when writing model to folder model_params after finished epoch 0 first 1000 steps. It's not a big issue but wasted several hours since I was not aware of that. Please add check (and create that folder in case of missing) in train.py. Thanks!

**Comment by nicolettiant89** [comment]
Same problem!

**Comment by violet-zct** [comment]
It worked for me.

2016-06-06 15:17 GMT+08:00 nicolettiant89 notifications@github.com:

**Comment by shagunsodhani** [comment]
Yes, I too had this issue and have raised a [PR](https://github.com/thomasmesnard/DeepMind-Teaching-Machines-to-Read-and-Comprehend/pull/10/commits) to fix this.
